index.js

```diff
@@ -134,83 +612,58 @@ app.intent('send.message - yes', async conv => {
 app.intent('send.message - no', async conv => {
   conv.contexts.delete('sendmessage_data');
   conv.ask('Message not sent. Is there anything else I can do for you?');
-  conv.ask(new Suggestions('No, that\'s all', 'Yes'));
+  conv.ask(new Suggestions("No, that's all", 'Yes'));
   conv.contexts.set('anything_else', 2);
 });
 
 /**
-* call.user
-*/
-app.intent('call.user', async (conv, {target}) => {
+ * call.user
+ */
+app.intent('call.user', async (conv, { target }) => {
   const circuit = await getCircuit(conv);
   if (!circuit) {
     return;
   }
 
-  let users = await circuit.searchUsers(target);
+  let users = await getUsers(circuit, conv, target);
 
-  if (!users.length) {
+  if (users.length !== 1) {
```

Comment:
Not sure why you changed this. Can you explain?

Reply:
The function <b>searchUsers</b> already checks for
```javascript
if (!users.length)
```
and
```javascript
if (users.length > 1)
```
so after the function is called, we just have to check if the function returned 1 result.